pkg/sfu/pacer/pacer.go

```diff
@@ -0,0 +1,31 @@
+package pacer
+
+import (
+	"time"
+
+	"github.com/pion/rtp"
+	"github.com/pion/webrtc/v3"
+)
+
+type ExtensionData struct {
+	ID      uint8
+	Payload []byte
+}
+
+type Packet struct {
+	Header             *rtp.Header
+	Extensions         []ExtensionData
+	Payload            []byte
+	AbsSendTimeExtID   uint8
+	TransportWideExtID uint8
+	WriteStream        webrtc.TrackLocalWriter
+	Metadata           interface{}
+	OnSent             func(md interface{}, sentHeader *rtp.Header, payloadSize int, sentTime time.Time)
+}
+
+type Pacer interface {
+	Enqueue(p Packet)
+	Stop()
```

Comment:
Simple interface to enqueue a packet to be sent and stop a pacer. At least one more method that will be needed is to set the target bitrate if we add a pacer that meters data out per the channel capacity.